### ISSUE TITLE
The line should stay on the center of the screen.

### ISSUE THREAD

**Issue by Etan31** [Etan31/UNO-offline-game#27]
This container should always stay at the middle of the screen regardless of number of cards.
```
    <div class="line">
      <img src="./assets/imgs/misc/line.png" alt="line">
    </div>
```
The heading with a player tag should always be positioned below and to the left side of the line.
`<h2 id="player2heading" class="player2">Player 2</h2>`

But for player1 should be positioned above and right side of the line.
`<h2 id="player1heading" class="player1">Player 1</h2>`

**Comment by briang17** [comment]
Hello, I'd like to work on this

**Comment by Etan31** [comment]
Okay, I'll assign them to you.

To test, here is my suggestion.
for player1: 
temporary comment this code to add more cards to the player1 deck of cards.
on the line: 626 script.js
```
    if (!playerTurn1) {
      alert("It's not your turn yet!");
      return;
    }
```

on player2 just drop a plus4 or plus2 card.

**Comment by briang17** [comment]
hello again @Etan31 ,  I just submitted a PR, kindly check it out :)

**Comment by Etan31** [comment]
I've merged your pr #28  to the main branch, but the player's tag is still shifting when many cards are on the player's deck of cards.
